rdr_service/offline/response_validation.py

```diff
@@ -113,7 +113,7 @@ def _send_error_message(self):
         self._output_result(result_text)
 
     def _insert_data(self):
-        for key in self._error_list.items():
+        for key, error_list in self._error_list.items():  # pylint: disable=unused-variable
```

Comment:
Do you think `for key in self._error_list.keys():` will be better here, so you don't have an unused variable?